index.js

```diff
@@ -21,25 +24,25 @@ module.exports = postcss.plugin('postcss-increase-specifity', function(options)
 		css.walkRules(function(rule) {
 			rule.selectors = rule.selectors.map(function(selector) {
 				// Apply it to the selector itself if the selector is a `root` level component
-				// `html:root:root:root`
+				// `html:not(#\\20):not(#\\20):not(#\\20)`
 				if(
 					selector === 'html' ||
-					selector === ':root' ||
+					selector === ':not(#\\20)' ||
 					selector === ':host' ||
 					selector === opts.stackableRoot
 				) {
 					return selector + opts.stackableRoot.repeat(opts.repeat);
 				}
 
 				// Otherwise just make it a descendant (this is what will happen most of the time)
-				// `:root:root:root .foo`
+				// `:not(#\\20):not(#\\20):not(#\\20) .foo`
 				return opts.stackableRoot.repeat(opts.repeat) + ' ' + selector;
 			});
 
 			if(opts.overrideIds) {
 				if(
 					// If an id is in there somewhere
-					(/#/).test(rule.selector) ||
+					(/#(?!\\)/).test(rule.selector) ||
```

Comment:
this thing needed to be changed, because otherwise original regexp catches `#` in `:not(#\20)` and doesnt add `!important`